client/app/scripts/charts/edge-container.js

```diff
@@ -50,14 +56,9 @@ class EdgeContainer extends React.Component {
   }
 
   render() {
-    const { layoutPrecision, points } = this.props;
+    const { layoutPrecision } = this.props;
     const other = _.omit(this.props, 'points');
 
-    if (layoutPrecision === 0) {
```

Comment:
That is true. But we had no way of knowing how many points there could be even with small topologies. The main goal was to not live in fear of this code blowing up. With your fix, it's robust.